src/Solver/Deterministic/DeDriver.cpp

```diff
@@ -203,47 +318,56 @@ void DeDriver::writeExtMps(const char * name)
 	assert(model_);
 
 	/** model info */
-	DspOsi * osi = NULL;
-	CoinPackedMatrix * mat = NULL;
-	double * clbd   = NULL;
-	double * cubd   = NULL;
-	double * obj    = NULL;
-	char *   ctype  = NULL;
-	double * rlbd   = NULL;
-	double * rubd   = NULL;
+	DspOsi *osi = NULL;
+	CoinPackedMatrix *mat = NULL;
+	double *clbd = NULL;
+	double *cubd = NULL;
+	double *obj = NULL;
+	CoinPackedMatrix *qobj = NULL;
+	char *ctype = NULL;
+	double *rlbd = NULL;
+	double *rubd = NULL;
 
 	BGN_TRY_CATCH
 
 	/** get DE model */
-	DSP_RTN_CHECK_THROW(model_->getFullModel(mat, clbd, cubd, ctype, obj, rlbd, rubd));
+	DSP_RTN_CHECK_THROW(model_->getFullModel(mat, clbd, cubd, ctype, obj, qobj, rlbd, rubd));
 
 	/** create DspOsi */
 	osi = createDspOsi();
-	if (!osi) throw CoinError("Failed to create DspOsi", "writeExtMps", "DeDriver");
+	if (!osi)
+		throw CoinError("Failed to create DspOsi", "writeExtMps", "DeDriver");
 
 	/** load problem */
 	osi->si_->loadProblem(*mat, clbd, cubd, obj, rlbd, rubd);
+	if (qobj != NULL)
+	{
+		osi->loadQuadraticObjective(*qobj);
+	}
+
```

Comment:
This can possibly be wrapped into `loadProblem`. Need to consider which one would look better.

Reply:
Let's leave this as it is for now.